lib/smart_answer_flows/marriage-abroad/outcomes/outcome_ceremonies_in_denmark_when_residing_in_uk_or_denmark.govspeak.erb

```diff
@@ -117,7 +117,7 @@
 
   <% end %>
 
-  <%= render partial: 'os_affirmation/fees.govspeak.erb',
+  <%= render partial: 'services_and_fees.govspeak.erb',
```

Comment:
Minor: It might be worth explaining in the commit note why you've added the `services` prefix to the partial name as well as moving it out of the `os_affirmation` sub-directory.